lib/index.js

```diff
@@ -56,6 +57,13 @@ class Sade {
 		return this;
 	}
 
+	alias(...names) {
+		if (this.single) throw new Error('Cannot call `alias()` in "single" mode');
+		if (!this.curr) throw new Error('Cannot call `alias()` before defining a command');
+		this.tree[this.curr].alibi = this.tree[this.curr].alibi.concat(...names);
```

Comment:
alibi? why calling it that instead of `alias` or `aliases`?

Reply:
Alias is already taken, for options. It's internal so it doesn't matter.

Reply:
ah yea... then swap them, haha.